x-pack/platform/plugins/shared/event_log/server/event_log_client.ts

```diff
@@ -207,6 +230,30 @@ export class EventLogClient implements IEventLogClient {
     await this.esContext.esAdapter.refreshIndex();
   }
 
+  public async findEventsBySavedObjectIdsSearchAfter(
```

Comment:
would it make more sense to just update the existing `findEventsBySavedObjectIds` function to handle the search_after options if they're passed in?